docs/guide/limitation_reference.md

```diff
@@ -402,6 +402,25 @@ is evaluated.
 For more information on how to restrict User's access to part of the Subtree 
 follow [the example in the Admin management section](admin_panel.md#restrict-editing-to-part-of-the-tree).
 
+## Version Lock Limitation
```

Comment:
Make sure that the Limitation does not need to be added to https://doc.ibexa.co/en/master/guide/limitations/#available-limitations